content/docs/api-reference/params_show.md

```diff
@@ -0,0 +1,253 @@
+# dvc.api.params_show()
+
+Returns all the <abbr>params</abbr> associated with the given <abbr>DVC
+repository</abbr>.
+
+```py
+def params_show(
+    *targets: str,
+    repo: Optional[str] = None,
+    stages: Optional[Union[str, Iterable[str]]] = None,
+    rev: Optional[str] = None,
+    deps: bool = False,
+) -> Dict:
+```
+
+#### Usage:
+
+```py
+import dvc.api
+
+params = dvc.api.params_show()
+```
+
+## Description
+
+Retrieves the <abbr>params</abbr> tracked in a <abbr>DVC repository</abbr>.
+
+Without arguments, this function will retrieve all <abbr>params</abbr> from all
+parameter files, for the current revision.
+
+See the options below to restrict the <abbr>params</abbr> retrieved.
+
+## Parameters
+
+- **`*targets`** (str, optional): Names of the parameter files to retrieve
```

Comment:
Please only bold required args, and replicate existing format in other pages. Thanks
```suggestion
- `*targets` - Names of the parameter files to retrieve e.g. "params.py, myparams.toml"
```
☝🏼 Just an idea. Please fix sample value somewhere in the paragraph, it's unclear what format this arg expects.